### PR TITLE
Fixing reviewed comments

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Fixes: https://github.com/wso2/carbon-apimgt/pull/8500#discussion_r555036725
Fixes: https://github.com/wso2/carbon-apimgt/pull/8500#discussion_r555038413
Fixes: https://github.com/wso2/carbon-apimgt/pull/8944#discussion_r554980478
Fixes: https://github.com/wso2/carbon-apimgt/pull/8984#discussion_r556263625
Fixes: https://github.com/wso2/carbon-apimgt/pull/9533#discussion_r559992241

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java [hunk 1]
```diff
@@ -10267,25 +10262,21 @@ private JSONObject getSecurityAuditConfigurationProperties(String tenantDomain)
         List<APIResource> removedReusedResources = new ArrayList<>();
 
         for (URITemplate existingUriTemplate : existingUriTemplates) {
-
             // If existing URITemplate is used by any API Products
             if (!existingUriTemplate.retrieveUsedByProducts().isEmpty()) {
                 String existingVerb = existingUriTemplate.getHTTPVerb();
                 String existingPath = existingUriTemplate.getUriTemplate();
                 boolean isReusedResourceRemoved = true;
```
Comment (praminda) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java:10269]: Its better to keep additional new line to separate out conditional blocks of the code
```suggestion
                boolean isReusedResourceRemoved = true;

```

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java [hunk 2]
```diff
@@ -10267,25 +10262,21 @@ private JSONObject getSecurityAuditConfigurationProperties(String tenantDomain)
...
-
                 for (URITemplate updatedUriTemplate : updatedUriTemplates) {
                     String updatedVerb = updatedUriTemplate.getHTTPVerb();
                     String updatedPath = updatedUriTemplate.getUriTemplate();
```
Comment (praminda) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java:10272]:
```suggestion
                    String updatedPath = updatedUriTemplate.getUriTemplate();

```

Comment (wasuradananjith) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java]: Fixed via https://github.com/wso2/carbon-apimgt/pull/9604/commits/81fdfdcc985a2429bec9fcfa553704b98a3b6b87

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java (hunk 1 above)
Comment (wasuradananjith) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java]: Fixed via https://github.com/wso2/carbon-apimgt/pull/9604/commits/81fdfdcc985a2429bec9fcfa553704b98a3b6b87

**Review by Meruja — APPROVED** [review]

**Review by vithu30 — APPROVED** [review]

**Review by menakaj — APPROVED** [review]

**Pull request merged**